bundles/pubsub/examples/pubsub/interceptors/src/ps_interceptor_activator.c

```diff
@@ -26,52 +26,54 @@
 struct interceptorActivator {
     first_interceptor_t *interceptor;
     uint64_t interceptorSvcId;
+    pubsub_interceptor_t *interceptorSvc;
 
     second_interceptor_t *secondInterceptor;
     uint64_t secondInterceptorSvcId;
+    pubsub_interceptor_t *secondInterceptorSvc;
```

Comment:
I know this is just a name change, but this does not need to be a pointer.
so:
```C
struct interceptorActivator {
     uint64_t interceptorSvcId;
     pubsub_interceptor_t interceptorSvc;
     uint64_t secondInterceptorSvcId;
     pubsub_interceptor_t secondInterceptorSvc;
};
```

should also works and save two malloc/callocs